renderer/VertexCache.cpp

```diff
@@ -18,207 +18,175 @@ Project: The Dark Mod (http://www.thedarkmod.com/)
 ******************************************************************************/
 
 #include "precompiled.h"
-#include "../game/DarkModGlobals.h"
 #include <mutex>
 #pragma hdrstop
 
 #include "tr_local.h"
 
-static const int	FRAME_MEMORY_BYTES = 0x200000;
-static const int	EXPAND_HEADERS = 32;
-
-idCVar idVertexCache::r_showVertexCache( "r_showVertexCache", "0", CVAR_INTEGER | CVAR_RENDERER, "" );
-idCVar idVertexCache::r_useMapBufferRange( "r_useMapBufferRange", "0", CVAR_BOOL | CVAR_RENDERER | CVAR_ARCHIVE, "use ARB_map_buffer_range for optimization" );
-idCVar r_queueVertexUploads( "r_queueVertexUploads", "0", CVAR_BOOL | CVAR_RENDERER | CVAR_ARCHIVE, "queue some vertex uploads from frontend to backend" );
+idCVar idVertexCache::r_showVertexCache( "r_showVertexCache", "0", CVAR_INTEGER | CVAR_RENDERER, "Show VertexCache usage statistics" );
+idCVar idVertexCache::r_staticVertexMemory( "r_staticVertexMemory", "40960", CVAR_INTEGER | CVAR_RENDERER, "Amount of static vertex memory, in kB (max 65535)" );
```

Comment:
I made the cache memory sizes adjustable so that we don't have to set them to an initial high (save) value. But if someone does hit the limit, they at least have the possibility to increase it.